curriculum/challenges/english/25-front-end-development/workshop-reusable-profile-card-component/674ef11f75254548672d998c.md

```diff
@@ -0,0 +1,124 @@
+---
+id: 674ef11f75254548672d998c
+title: Step 1
+challengeType: 0
+dashedName: step-1
+---
+
+# --description--
+
+In this project, you will learn how to work with props by building a reusable profile card component.
+
+Start by using the export keyword to create a `Card` functional component with `name`, `title`, and `bio` as the props. Don't forget to make sure they're all wrapped in a curly brace. That way, you destructure them instead of accessing them from `props`.
+
+Also, return a pair of parenthesis with empty string inside the function.
+
+# --hints--
+
+You should create a `Card` functional component. Don't forget to return an empty string inside it.
```

Comment:
This test doesn't work without the `export`, right? Maybe it's worth to remember that the component needs to be exported in this hint.

```suggestion
You should create and export a `Card` functional component. Don't forget to return an empty string inside it.
```